public/app/plugins/datasource/loki/LanguageProvider.test.ts

```diff
@@ -171,7 +171,7 @@ describe('Language completion provider', () => {
       const labelValues = await provider.fetchLabelValues('testkey', { streamSelector: '{foo="bar"}' });
       expect(requestSpy).toHaveBeenCalledWith('label/testkey/values', {
         end: 1560163909000,
-        query: '%7Bfoo%3D%22bar%22%7D',
+        query: '{foo="bar"}',
```

Comment:
I would prefer if we didn't change the encoding in this PR as it is not related to the changes, and if you'd like to do it, then change it in separate PR.